subsys/bluetooth/host/classic/Kconfig

```diff
@@ -178,6 +178,30 @@ config BT_A2DP_SINK
 
 endif # BT_A2DP
 
+config BT_AVCTP
+	bool "Bluetooth AVCTP protocol support"
+	help
+	  This option enables Bluetooth AVCTP support
+
+config BT_AVRCP
+	bool "Bluetooth AVRCP Profile [EXPERIMENTAL]"
+	select BT_AVCTP
+	select EXPERIMENTAL
+	help
+	  This option enables the AVRCP profile
+
+config BT_AVRCP_TARGET
+	bool "Bluetooth AVRCP Profile Target Function"
+	depends on BT_AVRCP
+	help
+	  This option enables the AVRCP profile target function
+
+config BT_AVRCP_CONTROLLER
+	bool "Bluetooth AVRCP Profile Controller Function"
+	depends on BT_AVRCP
+	help
+	  This option enables the AVRCP profile controller function
```

Comment:
I'd suggest removing the `depends on BT_AVRCP` and put these instead inside a `if BT_AVRCP` ... `endif` block. That way you can add more `BT_AVRCP_*` options in the future without having to declare `depends on BT_AVRCP` every time.

Reply:
Done.